tests/performance_tests/performance_test_basic.py

```diff
@@ -97,9 +215,11 @@ def checkTotalTrx():
     testGenerationDurationSec = 60
     targetTps = 1
     transactionsSent = testGenerationDurationSec * targetTps
+    cont = chainsContainer()
 
-    waitForEmptyBlocks(node1)
-    setupTotal = checkTotalTrx()
+    # Get stats prior to transaction generation
+    cont.startBlock = waitForEmptyBlocks(node1)
```

Comment:
I realize the variable naming was already in place, but would it be good to rename/identify a node as either a producer or validation node?  I think this would make understanding the context of the test, which node type is used for which purpose, more easily discernable.